frontend/src/app/projects/models/model-detail/model-detail.component.ts

```diff
@@ -41,17 +40,13 @@ export class ModelDetailComponent implements OnInit, OnDestroy {
       (gitModels) => (this.gitModels = gitModels)
     );
 
-    this._t4cModels.pipe(filter(Boolean)).subscribe((models) => {
-      this.t4cModels = models;
-    });
-
     this.t4cModelsSubscription = this.t4cModelService
       .listT4CModels(
         this.projectService.project!.slug,
         this.modelService.model!.slug
       )
       .subscribe((models) => {
-        this._t4cModels.next(models);
+        this.t4cModels = models;
       });
```

Comment:
I can't make a suggestion here (because of the deleted line), but I would change this to a simple one-liner `.subscribe((models) => (this.t4cModels = models));`